embassy-stm32/src/crc/mod.rs

```diff
@@ -0,0 +1,6 @@
+#[cfg_attr(crc_v2, path = "v2.rs")]
+#[cfg_attr(crc_v1, path = "v1.rs")]
+#[cfg_attr(crc_v3, path = "v2.rs")]
```

Comment:
it's somewhat confusing that `v3` uses `v2.rs`. Could you name it `v2v3.rs`? 

Also could you sort these attrs by version?